modules/start_on_team_color/functions/SOTC/fn_SOTC_Init.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+if (!isDedicated) then {
```

Comment:
Does not require !isdedicated check, this is executed only through clientInit

Reply:
Fixed.